### ISSUE TITLE
You don't really need to use "hotels/create" path. The fact that you are creating new resource is demonstrated by POST method usage.

### ISSUE THREAD

**Issue by vasekstebra** [VaclavStebra/pa165-booking-manager#81]

**Comment by Dominick1993** [comment]
@VaclavStebra was this issue connected to the REST API or Spring MVC? I "fixed" it in both cases and in other controllers (which are present in the MVC, not REST, so this was possibly connected to REST), even though I do not consider this a issue.

**Comment by vasekstebra** [comment]
I think this could also be fixed in REST  - see
```
@RequestMapping(value = "/create", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
produces = MediaType.APPLICATION_JSON_VALUE)
public final void createHotel(@RequestBody HotelDTO hotelDTO) throws Exception {
```

**Comment by Dominick1993** [comment]
As I mentioned above, I've fixed it in both cases 👍

**Comment by vasekstebra** [comment]
Sorry, I missed the REST controller in the pull request - my mistake :-)

**Comment by vasekstebra** [comment]
Fix in #83
